ComSemApp/student/views.py

```diff
@@ -8,11 +8,10 @@
 from django.contrib.auth.decorators import login_required
 from django.contrib.auth.decorators import user_passes_test
 from django.db.models import Q
-from django.views.generic import ListView, DetailView, CreateView, UpdateView
+from django.views.generic import ListView, DetailView, CreateView, UpdateView, TemplateView
```

Comment:
TemplateView is being used by the current implementation of pages as placeholders